### PR TITLE
fix iOS: use CGSize instead of NSSize. this is the correct type anyway

### PR DESCRIPTION
**Pull request by mtak-**
This is based on the work in this PR #60 by @Michael-Lfx which has not yet been accepted. I am guessing it's abandoned? If not feel free to reject this.

I, too, want this feature merged to help get `metal` (and ultimately `gfx`) building on iOS.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -28,14 +29,10 @@ use std::ops::Deref;
 use std::borrow::{Borrow, ToOwned};
 
 use objc::runtime::{Object, YES, NO};
-use cocoa::foundation::NSSize;
+pub use core_graphics::base::CGFloat;
```
Comment (grovesNL) [inline src/lib.rs]: @kvark do you have any strong preference about reexporting types?

**Comment by kvark** [comment]
Need to check the backend use of core_graphics. If it needs to depend on it directly anyway, then I'd rather not have metal-rs re-exporting those types.

**Comment by mtak-** [comment]
I've removed the reexports. gfx-metal-backend only needs `CGRect` and `CGFloat` from core_graphics. `CGRect` is not used by metal-rs so gfx-metal-backend would need core_graphics anyway.

**Comment by kvark** [comment]
Bors r+

**Comment by kvark** [comment]
@mtak- we'll publish 0.12 once #70 is merged

**Pull request merged**